### DISCUSSION TITLE
Hover states on phones

### DISCUSSION THREAD

**Discussion by tadassce** [tailwindlabs/tailwindcss#1998] [Ideas]
For the devices that don't have support for hover states (like phones and tablets), which can be detected using a media query:

```
@media (hover: none) { ... }
```

Should hover states be disabled on Tailwind? Or if not, what would you recommend as the best way of handling if I personally would like that behavior?

Thanks! 🙏

**Comment by estevanmaito** [comment]
One option is to do this: https://github.com/tailwindcss/tailwindcss/issues/1739#issuecomment-653751786
